static/assets/js/unsupported-browser.js

```diff
@@ -0,0 +1,16 @@
+window.parent.postMessage({msgType: "getVersionInfo"}, "*");
```

Comment:
For context, the extension will use this page as a full-screen iframe. It will reply to this specific message with the version number.
The buttons to export settings are only shown after we receive a reply with the version number.

The page continues to work even when visited normally, but without the ability to export settings.

Reply:
Put it directly on the comments if you wish